### PR TITLE
Adding Azure CGN

### PR DESCRIPTION
**Pull request by christianwaha**
Adding Azure Meetup Cologne

### CODE REVIEW

File: 2023/AzureCGN/data.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    ]
+  },
+  "CommunityInfo": {
+    "Name": "Azure Meetup Cologne",
+    "Country": "Germany",
+    "Url": "https://www.rakoellner.de/global-azure-cgn/",
+    "SessionizeApiId": "",
+    "Logo": {
+      "Description": "Community logo is a .png file 500X500px named {foldername}.png.",
+      "File": "_azurecgn-logo.png"
```
Comment (noopman) [inline 2023/AzureCGN/data.json:15]: Typo here in the file name, right? '_'

**Review by noopman — CHANGES_REQUESTED** [review]
Just a small typo as I can see it.

File: 2023/AzureCGN/data.json (hunk 1 above)
Comment (noopman) [reply 2023/AzureCGN/data.json:15]: *ping* @christianwaha!

**Review by noopman — APPROVED** [review]
Looks all clear now! :D

File: 2023/AzureCGN/data.json (hunk 1 above)
Comment (christianwaha) [reply 2023/AzureCGN/data.json]: Hi Magnus, yes seems that there is an underscore, which should be not there.

**Pull request merged**